pydantic/networks.py

```diff
@@ -325,7 +325,10 @@ def validate_email(value: str) -> tuple[str, str]:
             'value_error', 'value is not a valid email address: {reason}', {'reason': str(e.args[0])}
         ) from e
 
-    return name or parts['local'], parts['email']
+    email = parts.normalized
+    assert email is not None
+    name = name or parts.local_part
+    return name, email
```

Comment:
I'm curious, was this change necessary due to something in this PR? Seems unrelated lol

Reply:
I guess maybe it was a dependency bump thing

Reply:
Yeah. I accidentally bumped the locked version (because I recreated all lockfiles) and figured I might as well fix this so we can bump it before 2.0 and not have to support multiple versions. But I'm also happy to move that to another PR.

Reply:
I'm good with fixing it here, easiest to just do it in one go.